AnkiDroid/src/main/java/com/ichi2/anki/cardviewer/ViewerCommand.kt

```diff
@@ -71,7 +71,8 @@ enum class ViewerCommand(val resourceId: Int) {
     CHANGE_WHITEBOARD_PEN_COLOR(R.string.title_whiteboard_editor),
     SHOW_HINT(R.string.gesture_show_hint),
     SHOW_ALL_HINTS(R.string.gesture_show_all_hints),
-    ADD_NOTE(R.string.menu_add_note);
+    ADD_NOTE(R.string.menu_add_note),
+    RESCHEDULE_NOTE(R.string.reschedule_card_dialog_message);
```

Comment:
Just `Reschedule` reads better here
```suggestion
    RESCHEDULE_NOTE(R.string.card_editor_reschedule_card);
```